tests/networkpolicy_test.go

```diff
@@ -54,6 +54,7 @@ var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:compon
 		tests.PanicOnError(err)
 
```

Comment:
This commit description line is not clear to me: We first focus on dual stack and single stack lanes.
Do you mean to say that this commit only handles skipping NetworkPolicy tests on dual stack and single stack lanes?

Reply:
it only skip on `Kind`
single and dual are not skipped.

Reply:
ok but the wording "We first focus on dual stack and single stack lanes" is not very clear. 
could you reword this sentence?